src/Magnum/Math/DualQuaternion.h

```diff
@@ -39,6 +41,62 @@ namespace Implementation {
     template<class, class> struct DualQuaternionConverter;
 }
 
+/** @relatesalso Dual quaternion
+@brief Screw linear interpolation of two dual quaternions
+@param from   First quaternion
+@param to     Second quaternion
+@param t      Interpolation phase (from range @f$ [0; 1] @f$)
+
+Expects that the real parts of both dual quaternions are normalized. @f[
+\begin{array}{l}
+    {\hat q}_{ScLERP} = \hat p (\hat p^* \hat q)^t =
+        cos \left( t \frac {\hat a} 2 \right) + \hat {\boldsymbol n} \cdot sin \left( t \frac {\hat a} 2 \right) \\
+    \hat d = p^*q = [l, m] \\
+    \hat a = [\hat a_R, \hat a_D]; \hat a_R = 2 \cdot acos \left( l_S \right); \hat a_D = -2m_S \frac 1 {|l_V|} \\
+    \hat {\boldsymbol n} = [\hat l_V \frac 1 {|l_V|},
+        \left( m_V - \hat {\boldsymbol n}_R \frac {\hat a_D l_S} 2 \right)\frac 1 {|l_V|}]
+\end{array}
+@f]
+@see @ref Quaternion::isNormalized(), @ref slerp(const Quaternion<T>&, const Quaternion<T>&, T),
+    @ref lerp(const T&, const T&, U)
+ */
+template<class T> inline DualQuaternion<T> sclerp(const DualQuaternion<T>& from, const DualQuaternion<T>& to, const T t) {
+    CORRADE_ASSERT(from.real().isNormalized() && to.real().isNormalized(),
+        "Math::sclerp(): real parts of quaternions must be normalized", {});
+
+    const T dotResult = dot(from.real(), to.real());
+    /* Multiplying with -1.0f ensures shortest path when dot < 0. */
+    /* diff = \hat d = p^*q */
+    const DualQuaternion<T> diff = from.quaternionConjugated()*((dotResult < .0) ? to*-1.0f : to);
+
+    /* angle = t \hat a_D */
+    const T angle = acos(diff.real().scalar())*t;
+    /* precompute sin/cos for manifold use */
+    const T sinAngle = sin(angle);
+    const T cosAngle = cos(angle);
```

Comment:
Please use `std::` prefix to avoid accidental use of the (`double` only) C functions. (`std::acos`, `std::sin`, `std::cos`).